packages/crayons-extended/custom-objects/src/components/form-builder/components/field-editor.tsx

```diff
@@ -571,6 +571,10 @@ export class FieldEditor {
       }
     }
 
+    if (this.dataProvider.name === 'status') {
+      objValues['choices'] = this.dataProvider.choices;
```

Comment:
Can you spread these choices please, just to avoid reference issues, if any

Reply:
updated